internal/npm_install/ng_apf_library.js

```diff
@@ -0,0 +1,62 @@
+/**
```

Comment:
maybe this file would be better organized in the /internal/ng_apf_package folder? not sure if it should be grouped with "angular layering violations" code or "after installing node_modules" code

Reply:
I moved `ng_apf_library.js` from `npm_install` to `ng_apf_library`. Unfortunately this means the test script would have to simulate the behavior of generating the template by moving the file to the right place. Please see comments in `generate_build_file.js`, `generate_build_file.spec.js`, and `update_golden.js`.